scripts/ci/ensure-angular-dependencies-compatibility.js

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
@lurch I get the following error on OS X:

```sh
./scripts/ci/ensure-angular-dependencies-compatibility.sh: line 28: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
```

Not sure what's the equivalent to this `-A` option.